.circleci/config.yml

```diff
@@ -16,6 +16,12 @@ param_cli_cluster_backend: &param_cli_cluster_backend
     type: enum
     enum: ["docker"] # ["docker", "minikube"]
 
+# Install go version 1.18 to run our integration tests on the minimal version we support
+steps_install_go_1_18: &steps_install_go_1_18
+  - run: |
+      wget https://go.dev/dl/go1.18.10.linux-amd64.tar.gz
+      sudo rm -rf /usr/local/go
+      sudo tar -C /usr/local -xzf go1.18.10.linux-amd64.tar.gz
```

Comment:
our ubuntu_vm comes with go 1.17 by default. I could have upgraded the VM, but the version above this one comes with Go 1.20, and I figured the best is probably to test our code on the minimum version we support. So, installing it manually